src/screens/SearchScreen.js

```diff
@@ -39,18 +65,28 @@ export default function ({ navigation, props }) {
         </TouchableOpacity>
         <View style={styles.container}>
           <Ionicons style={styles.icon} name="calendar-outline" />
-          <TouchableOpacity
-            style={styles.calendar}
-            onPress={() => navigation.navigate("SearchLocation")}
-          >
-            <TextInput style={styles.text} placeholder="Tarih" />
+          <TouchableOpacity style={styles.calendar} onPress={() => setOpen(true)}>
+            <Text style={styles.text} onPress={() => setMyDateText(Date(setDate(date)))}>
```

Comment:
```suggestion
            <Text style={styles.text} onPress={() => setMyDateText(Date(setDate(date)).toLocaleString("tr-TR", {
  day: "numeric",
  month: "short",
  year: "numeric",
  hour: "numeric",
  minute: "2-digit"
}))}>
```
hizlica boyle bi cozum buldum.
ama olmasi gereken, burda onPress olmamasi. onun yerine bi useEffect ile her date degistiginde `myDateText` i formatlayip guncellemen